dist/scripts/cookie-consent.js

```diff
@@ -19,9 +25,10 @@
       return;
     }
 
-    var cookie_notice = document.getElementById('cookie-consent');
-    cookie_notice.removeAttribute('hidden');
-    document.body.classList.add('has-cookie-consent');
+    showCookieNotice();
+    var button = document.querySelector('[data-trigger-cookie-consent]');
+    console.log(button);
```

Comment:
debug code should be removed 😬 I also wonder why it's added in the dist script only 🤔 maybe added manually?

Reply:
fixed